packages/builder/test/fixture-projects/project-with-scripts/no-default-method-script.js

```diff
@@ -0,0 +1,3 @@
+
+const fs = require("fs")
+fs.appendFileSync("output.txt", "script with no default method has been loaded");
```

Comment:
for this kind of scripts let's do something simple (to speed up the tests) instead of loading packages. Eg:
```
1+2
```